database/communication.go

```diff
@@ -0,0 +1,69 @@
+package database
+
+import (
+	"context"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const getCommunications string = "Select id, name, subject, frequency_throttle, template from membership.communication;"
+const getCommunication string = "Select id, name, subject, frequency_throttle, template from membership.communication where name = $1;"
+const getLastCommunication string = "Select created_at from membership.communication_log where member_id = $1 and communication_id = $2;"
+const logCommunication string = "Insert into membership.communication_log (member_id, communication_Id) values ($1, $2);"
+
```

Comment:
Would it be alright if we could move this to a `DbMethod` file? It's so that we don't clutter our code in a single file.